temporal-sdk/src/main/java/io/temporal/internal/sync/WorkflowThreadContext.java

```diff
@@ -381,4 +383,14 @@ public Thread getCurrentThread() {
       lock.unlock();
     }
   }
+
+  /** Never throws, prerequisites should be checked by the calling code */
+  private void runLocked() {
+    inRunUntilBlocked = true;
```

Comment:
This name is now confusing or this assignment is a bit of a lie, as this could be from `destroy` now